mlflow/genai/judges/instructions_judge/__init__.py

```diff
@@ -158,11 +164,18 @@ def __call__(
                     )
                 template_values.update(expectations)
 
-            formatted_prompt = format_prompt(self._instructions, **template_values)
+            formatted_instructions = format_prompt(self._instructions, **template_values)
+
+            system_prompt_content = format_prompt(
+                INSTRUCTIONS_JUDGE_SYSTEM_PROMPT, instructions=formatted_instructions
+            )
+
+            full_prompt = system_prompt_content
+            full_prompt = add_output_format_instructions(full_prompt)
```

Comment:
@BenWilson2 I think we lost @alkispoly-db 's comment about how `format_prompt` should append a new `user` message to a chat history, rather than replacing template variables in a single-message prompt.

https://github.com/mlflow/mlflow/pull/17466#discussion_r2317031812

Reply:
Will add it back :)